rbac/internal/migrations/replicate_workspaces.py

```
@@ -217,3 +220,47 @@ def replicate_updated_workspaces(
+
+
+def replicate_deleted_workspaces(since: datetime.datetime, replicator: Optional[RelationReplicator] = None):
+    if replicator is None:
+        replicator = OutboxReplicator()
+
+    delete_logs = AuditLog.objects.filter(
+        created__gte=since, resource_type="workspace", action="delete", resource_uuid__isnull=False
+    )
+
+    for batch in itertools.batched(delete_logs.iterator(), 500):
+        _replicate_deleted_batch(replicator, [_DeletedWorkspaceEntry(l.tenant_id, l.resource_uuid) for l in batch])
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)